internal/hud/server/controller.go

```diff
@@ -28,8 +29,9 @@ type HeadsUpServerController struct {
 	initDone    bool
 }
 
-func ProvideHeadsUpServerController(port model.WebPort, hudServer *HeadsUpServer, assetServer assets.Server, webURL model.WebURL) *HeadsUpServerController {
+func ProvideHeadsUpServerController(noBrowser bool, port model.WebPort, hudServer *HeadsUpServer, assetServer assets.Server, webURL model.WebURL) *HeadsUpServerController {
```

Comment:
try running `make shorttest` so you can find all the test code that calls this function